README.md

```diff
@@ -1,2 +1,16 @@
 # JiujitsuMatch
 Jiujitsu Social Networking
```

Comment:
PLEASE FEEL FREE TO PUT IN YOUR CENTS. LOOKING FOR PARTNERS, FRIENDS, INVESTORS.